unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/Main.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using DCL.Components;
+using DCL.Helpers;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+namespace DCL
+{
+    public class Main : MonoBehaviour
```

Comment:
Can we add some basic XML documentation explaining the responsibilities of this Monobehaviour?

Reply:
done